interface/therapy_groups/therapy_groups_models/group_statuses_model.php

```diff
@@ -0,0 +1,58 @@
+<?php
```

Comment:
In concept it's a good idea but it causes problems with our loadModel in base_controller.php:
```
    protected function loadModel($name)
    {
        if(!isset($this->$name)){
            require(dirname(__FILE__) .'/../' . self::MODEL_FOLDER .'/'. strtolower($name) .'_model.php');
            $this->$name = new $name;
        }

        return $this->$name;
    }
```
This method loads both the class and the file using the namespace, so if we pass on the namespace as well, it will cause a problem here.
I'm not sure we can change this right now.